lib/private/Preview/Movie.php

```diff
@@ -53,23 +53,28 @@ public function getThumbnail(File $file, int $maxX, int $maxY): ?IImage {
 		}
 
 		$result = null;
-		if ($this->useTempFile($file)) {
-			// Try downloading 5 MB first, as it's likely that the first frames are present there.
-			// In some cases this doesn't work, for example when the moov atom is at the
-			// end of the file, so if it fails we fall back to getting the full file.
-			// Unless the file is not local (e.g. S3) as we do not want to download the whole (e.g. 37Gb) file
-			if ($file->getStorage()->isLocal()) {
-				$sizeAttempts = [5242880, null];
-			} else {
-				$sizeAttempts = [5242880];
-			}
-		} else {
-			// size is irrelevant, only attempt once
-			$sizeAttempts = [null];
-		}
+
+		// Timestamps to make attempts to generate a still
+		$timeAttempts = [5, 1, 0];
+
+		// By default, download $sizeAttempts from the file along with
+		// the 'moov' atom.
+		// Example bitrates in the higher range:
+		// 4K HDR H265 60 FPS = 75 Mbps = 9 MB per second needed for a still
+		// 1080p H265 30 FPS = 10 Mbps = 1.25 MB per second needed for a still
+		// 1080p H264 30 FPS = 16 Mbps = 2 MB per second needed for a still
+		$sizeAttempts = [10485760];
```

Comment:
```suggestion
		$sizeAttempts = [1024 * 1024 * 10];
```
Makes it much easier to understand the value